src/PixiEditor/Models/ExceptionHandling/CrashReport.cs

```diff
@@ -115,6 +116,8 @@ private static void AppendStateInfo(StringBuilder builder)
         builder
             .AppendLine("Environment:")
             .AppendLine($"  Thread Count: {GetFormatted(() => Process.GetCurrentProcess().Threads.Count)}")
+            .AppendLine("Analytics:")
```

Comment:
While we are at it, can you do anonymizing path in crash reports? It's better to make sure that everything is fully anonymous